src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -31,19 +31,14 @@
 
 import javax.annotation.Nullable;
 
+import com.ning.http.client.*;
```

Comment:
That is actually a hard one, because it is some kind of optimization from IntelliJ. When you import a certain number of classes from a package IntelliJ automatically replaces the imports by a wildcard one. But I managed to change it ;)